README.md

```diff
@@ -174,7 +174,10 @@ See example sketch.
 To be called after at least 5 minutes in fresh air.
 For v117: 0-65535 = automatic calibration.
 For v118: 0 = automatic calibration, 1-65535 manual calibration.
-- **bool getZeroCalibrationData(ZeroCalibrationData &data);** fills a data struct with the current zero calibration status and value.
+- **bool getZeroCalibrationData(ZeroCalibrationData &data)** fills a data struct with the current zero calibration status and value.
+Returns true on success.
+- **bool readRegister(uint8_t address, RegisterData &reg)** fills a data struct with the chip's register data at that address.
+Primarily intended for troubleshooting and analysis of the sensor. Not recommended to build applications on top of this method's raw data.
 Returns true on success.
```

Comment:
When true the struct is filled, false the data could not be read.

Reply:
Included your phrasing, plus added a note that this is atypical behaviour:
> Returns true when the struct is filled, false when the data could not be read.
Note: unlike other public methods, CRC errors don't return false or show up in `lastError()`, instead the CRC result is stored in `RegisterData.crcValid`.